src/style.css

```diff
@@ -1 +1,62 @@
-/* Styles go here */
+body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+
+.header {
+  position: relative;
+  max-width: 1200px;
+  margin: 0 auto;
+  height: 60px;
+}
+
+.container {
+  display: flex;
+  padding: 0 50px;
+}
+
+.header__img {
+  width: 40px;
+  height: 40px;
+  margin-top: 10px;
+}
+
+.box {
+  flex-grow: 1;
+}
+
+.nav {
+  padding-top: 19px;
+}
+
+.nav__link {
+  margin-left: 16px;
```

Comment:
Try to reset margin-left for first nav link

Reply:
https://github.com/mate-academy/layout_moyo-header/pull/2834/files#r1133067606